test/unit/errorResponseInterceptor.test.js

```diff
@@ -362,58 +344,54 @@ describe('test errorResponseInterceptor', () => {
     }
   });
 
-  it('test_added', async () => {
-    const expectedNumberOfResponseInterceptors = numberOfBaseInterceptors + 1;
+  /**
+   * Assert the error interceptors are present and correctly ordered.
+   *
+   * @returns the expected number of interceptors
+   */
+  function assertInterceptors() {
+    const expectedNumberOfResponseInterceptors = numberOfBaseInterceptors + 2;
```

Comment:
Can we make this 2 into const at top of the tests? CUSTOM_INTERCEPTORS_NUM or something? I'm afraid it is a bit subtle on a long run hidden here

Reply:
sure

Reply:
Fixed in ea62b7ab